webrender/src/texture_cache.rs

```diff
@@ -594,6 +600,42 @@ impl TextureCache {
         }
     }
 
+    pub fn insert_image_border_updating_operation(src: &[u8],
+                                                  alloc_x: u32,
+                                                  alloc_y: u32,
+                                                  alloc_width: u32,
+                                                  _alloc_height: u32,
+                                                  _request_x: u32,
+                                                  request_y: u32,
+                                                  request_width: u32,
+                                                  request_height: u32,
+                                                  stride: Option<u32>,
+                                                  bpp: u32,
+                                                  op: &mut BorderUpdatingMethod) {
```

Comment:
Let's pass this in as a closure. You only use it with temporary structs anyway, and using a closure would remove a lot of boilerplate.